layouts/partials/head.html

```diff
@@ -6,14 +6,14 @@
 <link rel="canonical" href="{{ .Permalink }}" />
 
 <!-- Theme CSS -->
-<link rel="stylesheet" href="/assets/style.css">
+<link rel="stylesheet" href="{{ .Site.BaseURL }}/assets/style.css">
```

Comment:
Could you use... `{{ $.Site.BaseURL }}` instead?
```suggestion
<link rel="stylesheet" href="{{ $.Site.BaseURL }}/assets/style.css">
```